src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js

```diff
@@ -70,7 +70,9 @@ const FishBeltTransectInputs = ({
   const tideOptions = [...getOptions(tides.data), { label: 'not reported', value: '' }]
   const fishbelt_transect = validationsApiData?.fishbelt_transect
   // account for empty starter row
-  const hasFishBeltObservations = observationsState.length > 1
+  const hasFishBeltObservations =
+    !!observationsState.length > 0 &&
+    Object.prototype.hasOwnProperty.call(observationsState[0], 'fish_attribute')
```

Comment:
doesn't feel that great - what if other observations have empty fish attributes? are we always going to be scanning the array of observations?

Reply:
hmmmm, ya, I am not sure about this proxy logic either....better for $$ and ux if we rethink the complicated disabled input thing.



Other than thaat, what do you think about `observationsState[0]?.fish_attribute` instead of `hasOwnProperty.call`?

Reply:
both could work, I would rather have it be true or false than throw an undefined error @mmnoo